otbox.py

```diff
@@ -44,23 +35,34 @@ class OtBox(object):
     PREFIX_CMD_HANDLER_NAME       = '_mqtt_handler_'
     OTBUX_SWTORUN_FILENAME        = 'otswtoload.json'
     LOCATION_FILE_NAME            = 'location.txt'
-    FIRMWARE_EUI64_RETRIEVAL      = '{0}{1}'.format(os.getcwd(),'/bootloaders/01bsp_eui64_prog.ihex')
-    FIRMWARE_TEMP                 = 'firmware_mote.ihex'
     NEW_SOFTWARE_FILE_NAME        = 'new_software.zip'
     PICTURE_FILENAME              = 'picture'
     INIT_PICTURE_URL              = 'https://upload.wikimedia.org/wikipedia/commons/7/74/Openwsn_logo.png'
 
     def __init__(self):
 
-        # store params
+        # parse args
+        parser = argparse.ArgumentParser()
+        parser.add_argument("--testbed", nargs="?", default="opentestbed")
```

Comment:
You should check user input, i.e. allow only `iotlab` and not `iot-lab`. When `otbox.py` is invoked with `--help`, list of expected values (supported testbeds) should be given.